packages/types/src/index.ts

```diff
@@ -333,6 +333,13 @@ export type Leg = {
   };
   to: Place;
   transitLeg: boolean;
+  trip?: {
```

Comment:
Can we make Trip its own type? And I'd like to include the full type from the GraphQL schema, but we can do that later in our OTP2/GraphQL complete refactor.

Reply:
Yes I'd prefer to do this in a future PR